utils/zerovec/src/map/kv.rs

```diff
@@ -17,10 +17,14 @@ use alloc::boxed::Box;
 #[allow(clippy::upper_case_acronyms)] // KV is not an acronym
 pub trait ZeroMapKV<'a> {
     /// The container that can be used with this type: [`ZeroVec`] or [`VarZeroVec`].
-    type Container: MutableZeroVecLike<'a, Self, GetType = Self::GetType, OwnedType = Self::OwnedType>
-        + Sized;
-    // TODO: Consider adding Slice here
-    // type Slice: ZeroVecLike<Self, GetType = Self::GetType> + ?Sized;
+    type Container: MutableZeroVecLike<
+            'a,
+            Self,
+            SliceVariant = Self::Slice,
+            GetType = Self::GetType,
+            OwnedType = Self::OwnedType,
+        > + Sized;
+    type Slice: ZeroVecLike<Self, GetType = Self::GetType> + ?Sized;
```

Comment:
If you don't like me adding the `Slice` type, it's easy to undo that part by reverting my last two commits.

Reply:
I'm fine with it. I really wish you could define type aliases in associated types though; these only exist so that the code using this is cleaner.